types/alias.go

```diff
@@ -40,6 +40,7 @@ const (
 	BombayChainID         = "bombay-12"
 	SwapDisableForkHeight = fork.SwapDisableForkHeight
 	SwapEnableForkHeight  = fork.SwapEnableForkHeight
+	OsmoIbcDenom          = "ibc/0ef15df2f02480ade0bb6e85d9ebb5daea2836d3860e9f97f9aade4f57a31aa0"
```

Comment:
I just saw this variable OsmoIbcDenom, is it not used some where?

Reply:
It is used in tests to test against the regex.